src/components/Pages/MainPage/MainPage.module.css

```
@@ -3,38 +3,27 @@
   width: 100%;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
-}
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
-}
-.logo.react:hover {
-  filter: drop-shadow(0 0 2em #61dafbaa);
-}
-
-@keyframes logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  a:nth-of-type(2) .logo {
-    animation: logo-spin infinite 20s linear;
-  }
+.MainPage {
+  display: flex;
+  flex-direction: row;
+  gap: 5%;
+  justify-content: center;
+  height: 100%;
+  padding: 30px;
 }
 
-.card {
-  padding: 2em;
+.MainPage .infoContainer {
+  backdrop-filter: blur(20px);
+  background: linear-gradient(151deg, rgba(255, 255, 255, 0.24) 0%, rgba(255, 255, 255, 0.06) 100%);
+  box-shadow: 0px 4px 24px -1px rgba(0, 0, 0, 0.2);
+  border-radius: 10px;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  padding: 32px;
 }
 
-.read-the-docs {
-  color: #888;
+.MainPage .tab {
+  color: white;
```

Would you mind using our white color that's set in the text variable?
The same goes for the other places where the color is set (in the `ProfileInformation` css for example).
```suggestion
  color: var(--text);
```